oklog/values.yaml

```diff
@@ -22,6 +22,9 @@ oklog:
       memory: 128Mi
       cpu: 125m
 
+  serviceType: LoadBalancer  # [ClusterIP|LoadBalancer]
```

Comment:
This is duplicated from service.type. May be a bit confusing. If we need an internal-lb service that exposes different ports let's make it a subsection. This is similar to how this code is structured: https://github.com/pinpt/worker/pull/172. Something like:

``` yaml
internalLBService: 
  enabled: false
  type: LoadBalancer
  ...
```
Defaults should be false. We can always use the agent to control whether to enable them or just pass flags in via the agent.